products/jhipster.md

```diff
@@ -0,0 +1,73 @@
+---
+title: JHipster
+category: app
+permalink: /jhipster
+iconSlug: NA
+versionCommand: yo jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
```

Comment:
Should be set to `false` to have only the `Supported` column (as requested by @captn3m0).